dataflux_pytorch/benchmark/README.md

```diff
@@ -18,42 +18,28 @@ First ensure you are running within a virtual python enviroment, make sure gclou
 gcloud config set project {PROJECT_ID}
 ```
 
-Then set the enviroment variables.
+Then set the command line variables.
 
-`CKPT_DIR_PATH` is the location of where to save the checkpoints. `STEPS` is the number of steps the model will take (the number of checkpoints created will be the same). The default value for `STEPS` is 5. The benchmark will run `save_checkpoint` repeatedly and produce the average at the end, then run `load_checkpoint` on all the saved checkpoints and produce the average.
+`--ckpt_dir_path` is the location of where to save the checkpoints. `--steps` is the number of steps the model will take (the number of checkpoints created will be the same). The default value for `--steps` is 5. The benchmark will run `save_checkpoint` repeatedly and produce the average at the end, then run `load_checkpoint` on all the saved checkpoints and produce the average.
```

Comment:
nit: it might be helpful to separate each flag on to its own line to make it easier to read and notice each flag that needs to be set